MSAL/src/instance/MSALAccountsProvider.m

```diff
@@ -182,6 +198,16 @@ - (MSALAccount *)accountForParameters:(MSALAccountEnumerationParameters *)parame
         return nil;
     }
     
+    [allAccounts addObjectsFromArray:msidAccounts];
+    [allAccounts addObjectsFromArray:brokerAccounts];
```

Comment:
Will this crash if brokerAccounts nil?

Reply:
It won't, but added if-conditions to avoid confusion.